src/api/endpoint/api/v1/token.ts

```diff
@@ -66,9 +65,14 @@ export default function(route: Router, auth: IAuth, storage: IStorageHandler, co
 				return next(ErrorCode.getCode(HTTP_STATUS.NOT_IMPLEMENTED, SUPPORT_ERRORS.STORAGE_NOT_IMPLEMENT));
 			}
 
+			// if JWT tokens are used enable the jwtid so we can revoke them
+			if (config.security?.api?.jwt?.sign) {
+				(config.security.api.jwt.sign as JWTSignOptions&{jwtid?:boolean}).jwtid = true;
```

Comment:
Could you share more context about `jwtid`?  I don't find anything clear enough.
https://github.com/auth0/node-jsonwebtoken

Reply:
the ```jwtid``` sets the ```jti``` claim in the token which is a unique identifier, see also:
https://tools.ietf.org/html/rfc7519#section-4.1.7